test/components-browser/fixtures/transclusion-rerender-stateful/test.js

```diff
@@ -5,17 +5,15 @@ module.exports = function(helpers) {
     var container = root.getComponent("container");
     var counter = root.getComponent("counter");
 
-    expect(document.body.textContent).to.equal("0");
+    expect(helpers.targetEl.textContent).to.equal("0");
```

Comment:
When hydrating, the body contains the hydration code as well as the `0`, so I've switched to `targetEl` instead.